doc/source/examples/rpt/tuning-parameters-with-nomad/tuning-parameters-with-nomad.rst

```diff
@@ -0,0 +1,320 @@
+- :math:`f_a(\alpha_j, \theta_j)` is the probability function of the non-interaction between the :math:`\gamma`-rays emitted within :math:`\Omega` and the material inside the vessel;
+- :math:`\Omega` is the closed exposed area of the detector, and
+- :math:`f_d(\alpha_j, \theta_j)` is the probability function of the interaction of the :math:`\gamma`-rays with the detector.
+
+The two probability functions mentioned above may be re-written the following way:
+
+.. math::
+
+    f_a(\alpha_j, \theta_j) = exp\{-\mu_r \ e(\alpha_j, \theta_j)\}
+
+where :math:`\mu_r` is the reactor's attenuation coefficient *(the third unknown parameter)* and :math:`e(\alpha_j, \theta_j)` is the length of the path traveled by the photon inside the reactor.
+
+And
+
+.. math::
+
+    f_d(\alpha_j, \theta_j) = 1 - exp\{ -\mu_d \ d(\alpha_j,\theta_j)\}
+
+where :math:`\mu_d` is the detector's attenuation coefficient and :math:`d(\alpha_j,\theta_j)` is the length of the path traveled by the photon inside the detector.
+
+
+:raw-html:`<br />`
+
+Parameter files
+----------------
+
+
+*rpt_parameters.prm* file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+RPT Parameters
+^^^^^^^^^^^^^^^^
+
+
+As seen in the previous example, in the subsection *"rpt parameters"*, we define the values of the set of parameter necessary for calculating the counts using the Monte Carlo method. These common parameters used for the RPT simulation are described in the `RPT parameters <../../../parameters/rpt/rpt_parameters.html>`_ documentation page.
+
+.. code-block:: text
+
+    # --------------------------------------------------
+    # RPT Monte Carlo technique
+    #---------------------------------------------------
+    subsection rpt parameters
+        set particle positions file          = positions.particle
+        set verbosity                        = quiet
+        set export counts                    = false
+        set counts file                      = run.csv
+        set monte carlo iteration            = 10000
+        set random number seed               = 0
+        set reactor height                   = 0.3
+        set reactor radius                   = 0.4
+        set peak-to-total ratio              = 0.4
+        set sampling time                    = 0.01
+        set gamma-rays emitted               = 2
+        set attenuation coefficient detector = 21.477
+    end
+
+.. attention::
+    ``verbosity`` **must** be set to **quiet** since NOMAD gets the cost function value from the terminal for its MADS algorithm.
+
+
+Parameter tuning
+^^^^^^^^^^^^^^^^^^
+
+In the subsection *"parameter tuning"*, we enable parameters tuning, we specify a type of cost function and define a set of artificial counts to compare with calculated counts. Parameters used for the tuning of the model parameters are described in the `Parameter tuning <../../../parameters/rpt/parameter_tuning.html>`_ documentation page.
```

Comment:
Why a set of artificial counts? We use a set of experimental counts.

Reply:
You're right, this was from before!